ui/package.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
For visibility https://github.com/ARC-Summer-2020/AutoRockClimber/pull/11#discussion_r479696332